<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>731</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="button_join_group">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>加入组播</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_leave_group">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>退出组播</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_clear">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清空文本</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_quit">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>69</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>组播地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="multiple_ip">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>69</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>239.0.0.1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="message">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>531</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_multicast_message">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>广播消息</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="multiple_port">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>71</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>65500</number>
     </property>
     <property name="value">
      <number>1200</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>组播端口</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>备注：组播地址范围为239.0.0.0-239.255.255.255，不能使用本机地址作为组播地址</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>731</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
